src/elasticsearch/config.go

```diff
@@ -33,6 +41,13 @@ func NewConfig() Config {
 			backoff = d
 		}
 	}
+	timeSuffix := TimeSuffixDay
+	if suffix := os.Getenv("ES_TIME_SUFFIX"); suffix != "" {
```

Comment:
What should be the behavior in case a user passes an unexpected value or typo? I think that in this case there would be a "silent ignore", not sure this is the best one. I think you should use `LookupEnv` instead, assign the default suffix (by day) if not set, switch on either `hour` or `day` and throw an error otherwise

Reply:
In this case, I guess we should return an error up, until `main()`. This includes returning errors for other config problems, such as `ES_BULK_TIMEOUT` not being a valid `time.Duration`. Maybe we should do this in another PR. WDYT?

Reply:
Up to you, maybe we should meet in the middle and at least `panic` the application for now to avoid weird behavior, or maybe log that it is defaulting to hour. But I agree that this change can be left to another PR

Reply:
Does panicking make sense given that this is not a required config? I guess we should log that we're using the default value(day) and move on. WDYT? @mmfrb @gppeixoto

Reply:
even for an optional config it's easy to check with `LookupEnv`, something like this should do it
```suggestion
	if ok, suffix := os.LookupEnv("ES_TIME_SUFFIX"); ok && suffix != "" {
```

then again you'd have to set it somewhere else the default value. up to you, you can move on with the logging for now as long as we leave an issue open for this and fix it on another PR appropriately